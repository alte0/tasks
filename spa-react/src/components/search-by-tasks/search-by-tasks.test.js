import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import SearchByTasks from './search-by-tasks';

it('Container render', () => {
    const tree = renderer
        .create(
            <Router>
                <SearchByTasks textSearch={'Text search data'} />)
            </Router>
        )
        .toJSON();

    expect(tree).toMatchSnapshot();
});
