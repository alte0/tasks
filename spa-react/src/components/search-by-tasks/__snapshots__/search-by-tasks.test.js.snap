// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container render 1`] = `
Array [
  <form
    className="form form_search"
    method="get"
    onSubmit={[Function]}
  >
    <h3
      className="form__title visually-hidden"
    >
      Поиск по задачам.
    </h3>
    <div
      className="form__row form__row_search"
    >
      <input
        className="form__input form__input_search"
        name="search-field"
        onChange={[Function]}
        placeholder="Поиск по задачам"
        type="search"
        value="Text search data"
      />
      <button
        className="form__button submit"
        disabled={true}
        type="submit"
      >
        Найти
      </button>
    </div>
  </form>,
  ")",
]
`;
